src/Resources/app/administration/src/module/sw-extension/snippet/de-DE.json

```
@@ -1,7 +1,10 @@
 {
     "sw-extension": {
         "in-app-purchase": {
-          "badge-label": "In-App Einkäufe"
+            "badge-label": "In-App Einkäufe",
+            "context-menu": {
+                "account-link-label": "In-App Einkäufe verwalten"
```

This is not the same as the English version.

```suggestion
                "account-link-label": "Einkäufe verwalten"
```